scripts/download_blenderkit.py

```diff
@@ -6,72 +6,80 @@
     raise Exception("This script only works with python3.x!")
 
 import urllib.request, json
-from urllib.request import urlretrieve, build_opener, install_opener
+from urllib.request import urlretrieve, build_opener, install_opener, urlopen
 from pathlib import Path
 import uuid
 
-assets = []
-page = 1
-while True:
-    print("Download metadata: page {}".format(page))
+def download_blendkit_assets(asset_types):
+    assets = {}
+    for asset_type in asset_types:
+        max_pages = 10
```

Comment:
>  It was previously while True: which while downloading all assets takes unspecified amount of time so a limit is convenient to have a controlled situation rather than looping infinitely only until an exception occurs.

I think we should still download all free assets. We want to have as many as possible and there is no real use-case where you only want 10 objects.

Reply:
To be correct only 9 objects, because you start at 1.